sklearn/utils/_pprint.py

```diff
@@ -4,62 +4,6 @@
 # Authors: The scikit-learn developers
 # SPDX-License-Identifier: BSD-3-Clause
 
-# Copyright (c) 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010,
```

Comment:
I don't know where this comes from but I guess it's kind of the same, we copied and pasted it from another place and we should keep the original licence?

Reply:
Yes, but then one of the previous merges added that the scikit-learn developers were the authors ...